google-cloud-bigquery/lib/google/cloud/bigquery/table.rb

```diff
@@ -1217,6 +1227,13 @@ def to_gapi
             @gapi
           end
 
+          def time_partitioning type: "DAY", expiration_ms: nil
```

Comment:
@jeremywadsack After reviewing some precedents, such as the [Website properties in Bucket](https://github.com/GoogleCloudPlatform/google-cloud-ruby/blob/master/google-cloud-storage/lib/google/cloud/storage/bucket.rb#L277-L310) and the handling of the [TimePartition properties in google-cloud-dotnet BigQuery](https://github.com/GoogleCloudPlatform/google-cloud-ruby/blob/master/google-cloud-storage/lib/google/cloud/storage/bucket.rb#L277-L310), we would prefer to go with flattened  properties, since there are only two. Also, I believe in google-cloud-ruby, our times are generally expressed as seconds, not ms.

```ruby
bigquery = Google::Cloud::Bigquery.new
dataset = bigquery.dataset "my_dataset"
table = dataset.create_table "my_table" do |table|
  table.time_patitioning_type = "DAY"
  table.time_patitioning_expiration = 86_400 #seconds
end
```

Does this sound OK?